src/components/ControlPlanes/ConnectButton.tsx

```diff
@@ -102,9 +102,12 @@ export default function ConnectButton(props: Props) {
             data-target={`/mcp/projects/${props.projectName}/workspaces/${extractWorkspaceNameFromNamespace(
               props.workspaceName,
             )}/mcps/${props.controlPlaneName}/context/${context.name}`}
-            additionalText={
-              currentContext === context.name ? '(default IdP)' : undefined
-            }
+            additionalText={`(${
```

Comment:
please use the mcp resource and check the `spec.authentication.enableSystemIdentityProvider === true`

Reply:
Changed as discussed